### PR TITLE
feat: dittohq.xyz sepolia deployment addresses

### PR DESCRIPTION
**Pull request by relyt29**
New contract addresses for sepolia, since opensea and reservoir are dropping support for goerli

### CODE REVIEW

**Comment by georgeroman** [comment]
@relyt29 Can you lowercase all addresses?

**Comment by relyt29** [comment]
yup, sorry forgot you guys use lowcase

File: packages/sdk/src/ditto/addresses.ts [hunk 1]
```diff
@@ -3,34 +3,41 @@ import { ChainIdToAddress, Network } from "../utils";
...
 export const Test20: ChainIdToAddress = {
   [Network.Ethereum]: "0x0000000000000000000000000000000000000000",
   [Network.EthereumGoerli]: "0x8caa8de40048c4c840014bdec44373548b61568d",
+  [Network.EthereumSepolia]: "0x22b4335feebc4c1dba508bbf96a7c0379dc2d06f",
 };
 
 export const UpshotOracle: ChainIdToAddress = {
   [Network.Ethereum]: "0xab484060281f073eed1519a7899cbde4b1af4695",
   [Network.EthereumGoerli]: "0x6c9808cb1fb906f34d9ab96b7b58ef91a67062b5",
+  [Network.EthereumSepolia]: "0x76cd8149d9280258682C55eae44194424A68bFa0",
```
Comment (georgeroman) [inline packages/sdk/src/ditto/addresses.ts:42]: Sorry, this is still mixed-case :)

Comment (relyt29) [reply packages/sdk/src/ditto/addresses.ts:42]: you have sharp eyes, don't know how I missed that one

**Pull request merged**